src/Analysis/GWASV2/Shared/AttritionTableWrapper/AttritionTable/AttritionTableRow.jsx

```diff
@@ -111,7 +112,7 @@ const AttritionTableRow = ({
     if (rowType === 'Outcome') {
       return outcome.variable_type === 'concept'
         ? outcome.concept_name
-        : 'this is for case control, TODO';
+        : 'this is for case control, TODO!';
```

Comment:
Not sure what needs to be done here: Can we just output outcome.concept_name if there is not a outcome.variable_type? This returns "this is for case control, TODO!" when the outcome object is missing the variable_type key.